### PR TITLE
NONE: Add service central client GetService test

### PR DESCRIPTION
**Pull request by fcobb**
Adds a basic happy path test for one of the main service central calls.
Also fixes a minor spelling mistake: `ReqestSnapshots` -> `RequestSnapshots`

### CODE REVIEW

File: pkg/servicecentral/client_test.go [hunk 1]
```diff
@@ -246,6 +247,28 @@ func TestGetServiceDataFailsWhenServiceCentralInternalError(t *testing.T) {
...
+		JSONFromFile(t, "get_service.rsp.json"),
+	))
+	serviceCentralServerMock := httptest.NewServer(handler)
+	defer serviceCentralServerMock.Close()
+	// when
+	serviceCentralClient := testServiceCentralClient(t, serviceCentralServerMock.URL, pkitest.MockASAPClientConfig(t))
+	_, err := serviceCentralClient.GetService(context.Background(), optionalUser, string(testServiceName))
+
+	// then
+	assert.NoError(t, err)
```
Comment (ash2k) [inline pkg/servicecentral/client_test.go]: Can be removed - see next line

File: pkg/servicecentral/testdata/get_service.rsp.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "data": [
```
Comment (fcobb) [inline pkg/servicecentral/testdata/get_service.rsp.json]: This is taken directly from the sample from the API docs.

File: pkg/servicecentral/client_test.go [hunk 3]
```diff
@@ -246,6 +247,28 @@ func TestGetServiceDataFailsWhenServiceCentralInternalError(t *testing.T) {
 	assert.True(t, httputil.IsUnknown(err))
 }
 
+func TestGetService(t *testing.T) {
```
Comment (fcobb) [inline pkg/servicecentral/client_test.go]: This structure was mostly copied from similar test in this file.

File: pkg/util/httputil/httptest/mock.go [hunk 4]
```diff
@@ -6,15 +6,15 @@ import (
 )
 
 type HTTPMock struct {
-	handlers        []http.Handler
-	ReqestSnapshots *RequestSnapshotter
+	handlers         []http.Handler
+	RequestSnapshots *RequestSnapshotter
```
Comment (fcobb) [inline pkg/util/httputil/httptest/mock.go]: Here is where I made the spelling fix.

**Pull request merged**